app/javascript/notifications.vue

```diff
@@ -1,72 +1,167 @@
+          :page-range="5"
+          :prev-text="`<i class='fas fa-angle-left'></i>`"
+          :next-text="`<i class='fas fa-angle-right'></i>`"
+          :first-button-text="`<i class='fas fa-angle-double-left'></i>`"
+          :last-button-text="`<i class='fas fa-angle-double-right'></i>`"
+          :click-handler="paginateClickCallback"
+          :container-class="'pagination__items'"
+          :page-class="'pagination__item'"
+          :page-link-class="'pagination__item-link'"
+          :disabled-class="'pager-disable'"
+          :active-class="'is-active'"
+          :prev-class="'is-prev pagination__item'"
+          :prev-link-class="'is-prev pagination__item-link'"
+          :next-class="'is-next pagination__item'"
+          :next-link-class="'is-next pagination__item-link'"
+          :first-last-button="true"
+          :hide-prev-next="true"
+          :margin-pages="0"
+          :break-view-text=null
+        )
+  .container(v-else-if="loaded")
+    .o-empty-massage
+      .o-empty-massage__icon
+        i.far.fa-smile
+      p.o-empty-massage__text(v-if='isUnreadPage')
+        | 未読の通知はありません
+      p.o-empty-massage__text(v-else)
+        | 通知はありません
+  .container(v-else)
+    | ロード中
 </template>
+
 <script>
+import Notification from './notification.vue'
+import VueJsPaginate from "vuejs-paginate"
+import UnconfirmedLinksOpenButton from './unconfirmed_links_open_button'
+
 export default {
+  props: {
+    mentorLogin: {
+      type: Boolean
+    }
+  },
+  components: {
+    'notification': Notification,
+    'pager-top': VueJsPaginate,
+    'pager-bottom': VueJsPaginate,
+    'unconfirmed-links-open-button': UnconfirmedLinksOpenButton
+  },
   data: () => {
     return {
-      notifications: []
+      notifications: [],
+      totalPages: 0,
+      currentPage: null,
+      loaded: false
     }
   },
   created() {
-    fetch(`/api/notifications.json`, {
-      method: 'GET',
-      headers: {
-        'X-Requested-With': 'XMLHttpRequest',
-      },
-      credentials: 'same-origin',
-      redirect: 'manual'
-    })
-      .then(response => {
-        return response.json()
-      })
-      .then(json => {
-        json.forEach(n => { this.notifications.push(n) })
-      })
-      .catch(error => {
-        console.warn('Failed to parsing', error)
-      })
+    // ブラウザバック・フォワードした時に画面を読み込ませる
+    window.onpopstate=function(){
+      location.href = location.href
+    }
+    this.currentPage = Number(this.getPageValueFromParameter()) || 1
+    this.getNotificationsPerPage()
   },
-  methods: {
-    clickBell() {
-      if (!this.notificationExist) {
-        location.href = '/notifications'
+  computed: {
+    url() {
+      if(this.isUnreadPage){
+        return `/api/notifications/unread.json?page=${this.currentPage}`
+      }else{
+        return `/api/notifications.json?page=${this.currentPage}`
       }
+    },
+    isUnreadPage(){
+      return location.pathname.includes('unread') ? true : false
```

Comment:
`includes`の返り値は真偽値なので、以下のように書けそうです。
```suggestion
      return location.pathname.includes('unread')
```